src/NITTA/Frontends/XMILE/Frontend.hs

```diff
@@ -37,6 +37,7 @@ data XMILEAlgBuilder v x = XMILEAlgBuilder
     , algDefaultValues :: HM.HashMap T.Text Double
     , algNextFreeNameIndex :: HM.HashMap T.Text Int
     , algNextArgIndex :: Int
+    , algAliases :: HM.HashMap T.Text T.Text
```

Comment:
Add docs about what it is

Reply:
Also, can you describe in the module header logic of transpilation. E.g., how names can change from XMILE to DF?